### PR TITLE
feat(docs): post on `Durable Execution`.

### PR DESCRIPTION
**Pull request by destifo**
## A blog about durable execution

-

#### Migration notes

...

- [ ] The change comes with new or modified tests
- [ ] Hard-to-understand functions have explanatory comments
- [ ] End-user documentation is updated to reflect the change

## Summary by CodeRabbit

- **New Features**
  - Introduced a comprehensive guide on distributed execution flow paradigms, focusing on managing interconnected processes in cloud development.
  - Added practical examples for implementing temporal workflows in order delivery systems.

- **Improvements**
  - Updated platform compatibility to support aarch64 architecture.
  - Enhanced dependency management with updated identifiers and versioning for modules.
  - Expanded configuration options for typegates to include temporal settings.

- **Documentation**
  - Added detailed explanations of durable execution and distributed execution patterns, along with practical insights for implementation.

### CODE REVIEW

**Comment by destifo** [comment]
@coderabbitai review

**Comment by destifo** [comment]
@coderabbitai review

File: website/blog/2024-08-13-durable-execution/index.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- Example in metatype
+
+# Questions
+- What are we improving here, or what makes us different, what are we enabling
+
+```
+
+# Integrating Durable Execution into existing systems.
+
+In this age of cloud development and microservices architecture, problems start to arise with the amounts of workloads that run in the system. **ADD Real World Example Here** Having multiple components in your system introduces more failure points and that's a common phenomenon in complex systems. But one important behavior of any application which the business side expects is that for an execution flow to reach its completion. Especially as systems grow in feature and complexity, chances are high that long-running processes will start to appear in our systems. To ensure that these processes complete in the intended manner, solutions were introduced in the last few decades.
```
[the remark this answers is not in the document]
Comment (destifo) [reply website/blog/2024-08-13-durable-execution/index.mdx:24]: yes, generate a suitable real-world example that would make the blog interesting and build a great understanding of the problem statement.

**Review by michael-0acf4 — APPROVED** [review]

**Comment by destifo** [comment]
@coderabbitai review

File: website/blog/2024-08-13-distributed-execution-flow-paradigms/index.mdx [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+
+Below is an example of how you can build a simple API to interact with an order delivery temporal workflow within Metatype.
+
+:::note
+If you are new to Metatype or haven’t set it up yet in your development environment. You can follow this [guideline](/docs/tutorials/quick-start).
+:::
+
+For this example, the order delivery system will have few components/services such as Payment, Inventory and Delivery.
+
+Your temporal workflow definition should look similar to the one below.
```
Comment (Yohe-Am) [inline website/blog/2024-08-13-distributed-execution-flow-paradigms/index.mdx:141]: Ending with a large code-dump is highly unreadable. I suggest you break it down and intersperse it with explanations of what each section is trying to accomplish. You also don't really need to showcase it in two languages.

File: website/blog/2024-08-13-distributed-execution-flow-paradigms/index.mdx [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+import { Worker, WorkflowClient } from "@temporalio/client";
+import { proxyActivities } from "@temporalio/workflow";
+
+export const { processPayment, checkInventory, deliverOrder } =
+  proxyActivities<{
+    processPayment(orderId: string): Promise<string>;
+    checkInventory(orderId: string): Promise<string>;
+    deliverOrder(orderId: string): Promise<string>;
+  }>({
+    startToCloseTimeout: "10 seconds",
```
Comment (Yohe-Am) [inline website/blog/2024-08-13-distributed-execution-flow-paradigms/index.mdx:157]: You forgot to include the file with the activities I think.

File: website/blog/2024-08-13-distributed-execution-flow-paradigms/index.mdx [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+});
+```
+
+Down below is the workflow communication bridge for this system expressed within a [Typegraph](/docs/reference/typegraph).
+
+```typescript
+import { Policy, t, typegraph } from "@typegraph/sdk/index.ts";
+import { TemporalRuntime } from "@typegraph/sdk/providers/temporal.ts";
+
+function getEnvVariable(
```
Comment (Yohe-Am) [inline website/blog/2024-08-13-distributed-execution-flow-paradigms/index.mdx:189]: suggestion: You don't really need to show this. You can just use the node approach for the article.

**Review by Yohe-Am — CHANGES_REQUESTED** [review]
Ah, I'm realizing this is still a work in progress.

**Comment by destifo** [comment]
@coderabbitai review

File: examples/metatype.yaml [hunk 5]
```diff
@@ -70,6 +70,9 @@ typegates:
         BASIC_andim: hunter2
       func-gql:
         POSTGRES: "postgresql://postgres:password@localhost:5432/db?schema=func_gql"
+      temporal:
```
Comment (Natoandro) [inline examples/metatype.yaml:73]: What about `prd`?

**Review by Natoandro — APPROVED** [review]
Great !

File: examples/metatype.yaml (hunk 5 above)
Comment (destifo) [reply examples/metatype.yaml:73]: the `temporal` typegraph is excluded so I don't think it's necessary, I only added this for testing purposes when manually deploying the `temporal` typegraph.

**Review by Yohe-Am — APPROVED** [review]

**Pull request merged**